1-download_sra.R

```diff
@@ -0,0 +1,50 @@
+# CCDL ALSF 2018
+# C. Savonen
+#
+# Purpose: This script is used in a bash loop to download file(s) from command line.
+#
+# Options:
+#  "-s" Run sample ID of the RNA-seq data you would like to download.
+#  "-d" Path to directory where you would like the data downloaded to go
+#  "-q" Path to downloaded metadata SQL file from GEO. Usually file is
+#       called "SRAmetadb.sqlite"
+#
+# Example usage in bash:
+#
+# Rscript scripts/1-download_sra.R \
+# -s SRR3934349 \
+# -q ref_files/SRAmetadb.sqlite \
+# -d raw_data
+
+# Using SRAdb to download the file and optparse to determine options from bash use
+library(optparse)
+library(SRAdb)
+
+# Get options from command line
+option_list <- list( 
+  make_option(opt_str = c("-s", "--sample"), type = "character",
```

Comment:
I think you should either remove the option for separating IDs by commas or tackle this a bit differently -- i.e., accept a file with accession codes for samples _or_ have this take study (`SRPXXXXXX`) accession codes as well. The commas bit seems too unwieldy to me.

Reply:
I don't really use the multiple files option in this pipeline. I purposely don't intend this to take SRP because the point of this is to download files individually or in batches so as to not download thousands of samples at once. Although as it is written, I'm fairly certain it could download the whole SRP if it was given it. Perhaps I should just get rid of the multiple files download, it is not used in this pipeline.

Reply:
I just got rid of that option because it doesn't even really fit with this pipeline.

Reply:
> I think you should either remove the option for separating IDs by commas

Fine with me.